Makefile

```
@@ -24,3 +24,61 @@ build-catalog:
 		--filter-by-os="linux/amd64" \
 		--from=registry.redhat.io/openshift4/ose-operator-registry:$(OCP_VERSION) \
 		--to=quay.io/3scaleops/olm-catalog:$(CATALOG_RELEASE)
+
+########################
+#### Bundle targets ####
+########################
+
+# find or download yq
+# download yq if necessary
+yq:
+ifeq (, $(shell command -v yq 2> /dev/null))
+	@GO111MODULE=off go get github.com/mikefarah/yq/v3
+YQ=$(GOBIN)/yq
+else
+YQ=$(shell command -v yq 2> /dev/null)
+endif
+
+REGISTRY = quay.io/3scaleops
+BUNDLE_CATALOG_DIR = olm-bundle-catalog
+BUNDLE_CATALOG_IMG = $(REGISTRY)/olm-catalog:bundle
+BASE_TMP_DIR = /tmp/3scaleops-olm-catalog
+PROJECT_NAME = $(notdir $(basename $(SOURCE_REPOSITORY)))
+TMP_DIR = $(BASE_TMP_DIR)/$(PROJECT_NAME)_$(PROJECT_VERSION)
+MANIFESTS_DST_DIR = $(BUNDLE_CATALOG_DIR)/$(PROJECT_NAME)/$(PROJECT_VERSION)
+BUNDLE_SUFFIX = threescale
+BUNDLE_IMG = $(REGISTRY)/$(PROJECT_NAME)-$(BUNDLE_SUFFIX)-bundle:$(PROJECT_VERSION)
+
+.PHONY: build-bundle-manifests
+# Example: make bundle-build SOURCE_REPOSITORY=https://github.com/3scale/marin3r.git PROJECT_VERSION=v0.7.0-alpha5
+bundle-manifests: clean
+	git clone --depth 1 --branch $(PROJECT_VERSION) $(SOURCE_REPOSITORY) $(TMP_DIR) 2> /dev/null
+	mkdir -p $(MANIFESTS_DST_DIR)
+	cp -a $(TMP_DIR)/bundle/* $(MANIFESTS_DST_DIR)
+	cp -a $(TMP_DIR)/bundle.Dockerfile $(MANIFESTS_DST_DIR)
+	sed -E -i 's@bundle/@@' $(MANIFESTS_DST_DIR)/bundle.Dockerfile
```

```suggestion
	sed -E -i '' 's@bundle/@@' $(MANIFESTS_DST_DIR)/bundle.Dockerfile
```

This command fails in mac if the inplace extension is not defined.